uint8 src_index
uint8 tar_index

float32 cur_vel
float32 cur_dist
float32 cur_angle

float32[] coef_a
float32[] coef_b
float32[] coef_c

